from great_expectations.exceptions import GreatExpectationsError


class ExecutionEngineError(GreatExpectationsError):
    pass
